Symfony/CS/Tests/Tokenizer/TokensTest.php

```diff
@@ -1061,4 +1062,88 @@ public function provideArrayExceptions()
 
         return $cases;
     }
+
+    /**
+     * @dataProvider getImportUseIndexesCases
+     */
+    public function testGetImportUseIndexes(array $expected, $input, $perNamespace = false)
+    {
+        $tokens = Tokens::fromCode($input);
+        $this->assertSame($expected, $tokens->getImportUseIndexes($perNamespace));
+    }
+
+    public function getImportUseIndexesCases()
+    {
+        $cases = array(
+            array(
+                array(1, 8),
+                '<?php use E\F?><?php use A\B;',
+            ),
+            array(
+                array(array(1), array(14), array(29)),
+'<?php
+use T\A;
+namespace A { use D\C; }
+namespace b { use D\C; }
+',
+                true,
+            ),
+            array(
+                array(array(1, 8)),
+                '<?php use D\B; use A\C?>',
+                true,
+            ),
+            array(
+                array(1, 8),
+                '<?php use D\B; use A\C?>',
+            ),
+            array(
+                array(7, 22),
+'<?php
+namespace A { use D\C; }
+namespace b { use D\C; }
+',
+            ),
+            array(
+                array(3, 10, 34, 45, 54, 59, 77, 95),
+ <<<'EOF'
+use Zoo\Bar;
+use Foo\Bar;
+use Foo\Zar\Baz;
+
+<?php
+
+use Foo\Bar;
+use Foo\Bar\Foo as Fooo, Foo\Bar\FooBar as FooBaz;
+ use Foo\Bir as FBB;
+use Foo\Zar\Baz;
+use SomeClass;
+   use Symfony\Annotation\Template, Symfony\Doctrine\Entities\Entity;
+use Zoo\Bar;
+
+$a = new someclass();
+
+use Zoo\Tar;
+
+class AnnotatedClass
+{
+}
+EOF
+                ,
+            ),
+        );
+
+        if (PHP_VERSION_ID >= 70000) {
```

Comment:
create new `testMethod` with different delivery provider, so this case will be marked as skipped on php 5.x instead of completely ignored